### PR TITLE
FIX: ISXB-927 Ignore platform-specific feature flags relating to Windows Gaming Input on all platforms. (Obsolete)

### PR DESCRIPTION
**Pull request by ekcoh**
### Description

Fix for ISXB-927 which ignore platform-specific feature flags relating to Windows Gaming Input when entering play-mode on non-Windows platforms.

### Changes made

~~Excludes code reading and sending IOCTL command when WGI feature flag is explicitly set on non-Windows platforms instead of generating error messages about failed IOCTL on platforms where there is no other possible outcome than failed IOCTL result.~~

Removes obsolete Windows Gaming Input feature flag that should have been removed already when original WGI support was added since it was superseded by `PlayerSettings.windowsGamepadBackendHint`

### Testing

No automated tests added at this point.

### Risk

Small risk.

### Checklist

Before review:

- [x] Changelog entry added.
    - Explains the change in `Changed`, `Fixed`, `Added` sections.
    - For API change contains an example snippet and/or migration example.
    - JIRA ticket linked, example ([case %<ID>%](https://issuetracker.unity3d.com/product/unity/issues/guid/<ID>)). If it is a private issue, just add the case ID without a link.	
    - Jira port for the next release set as "Resolved".
- [ ] Tests added/changed, if applicable.
    - Functional tests `Area_CanDoX`, `Area_CanDoX_EvenIfYIsTheCase`, `Area_WhenIDoX_AndYHappens_ThisIsTheResult`.
    - Performance tests.
    - Integration tests.
- [ ] Docs for new/changed API's.
    - Xmldoc cross references are set correctly.
    - Added explanation how the API works.
    - Usage code examples added.
    - The manual is updated, if needed.

During merge:

- [ ] Commit message for squash-merge is prefixed with one of the list:
    - `NEW: ___`.
    - `FIX: ___`.
    - `DOCS: ___`.
    - `CHANGE: ___`.
    - `RELEASE: 1.1.0-preview.3`.

After merge:

- [ ] Create forward/backward port if needed. If you are blocked from creating a forward port now please add a task to ISX-1444.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/InputManager.cs [hunk 1]
```diff
@@ -2645,12 +2645,15 @@ internal void ApplySettings()
                 runPlayerUpdatesInEditMode = m_Settings.IsFeatureEnabled(InputFeatureNames.kRunPlayerUpdatesInEditMode);
                 #endif
 
+                // Ignore
```
Comment (ritamerkl) [inline Packages/com.unity.inputsystem/InputSystem/InputManager.cs]: I would suggest to remove the comment or change it to say something like "Prevent error messages - don't run when WGI feature flag is explicitly set on non-Windows platforms"

**Review by ritamerkl — APPROVED** [review]
Looks good, I did not test it though, I think it is sufficient if Paulius approves.

File: Packages/com.unity.inputsystem/InputSystem/InputManager.cs (hunk 1 above)
Comment (ekcoh) [reply Packages/com.unity.inputsystem/InputSystem/InputManager.cs]: Adressed in last commit.

File: Packages/com.unity.inputsystem/InputSystem/InputManager.cs [hunk 2]
```diff
@@ -7,6 +7,7 @@
 using UnityEngine.InputSystem.Composites;
 using UnityEngine.InputSystem.Controls;
 using Unity.Collections.LowLevel.Unsafe;
+using UnityEditor;
```
Comment (ritamerkl) [inline Packages/com.unity.inputsystem/InputSystem/InputManager.cs]: Looks like accidentaly added?

Comment (ekcoh) [reply Packages/com.unity.inputsystem/InputSystem/InputManager.cs]: Thanks, fixed in latest commit

**Review by stefanunity — APPROVED** [review]

**Pull request merged**